src/client/styles/style.scss

```
@@ -59,7 +55,7 @@ body {
 .pic {
     width: 400px;
     height: 300px;
-    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+	box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.25);
```

You can omit `px` size specifier
```suggestion
	box-shadow: 0 0 10px rgba(0, 0, 0, 0.25);
```